packs/pagerduty/README.md

```diff
@@ -1,25 +1,31 @@
 # PagerDuty 
-This action enables the integration of PD into stackstorm. It is capable of performing the following actions
+
+This action enables the integration of PagerDuty into StackStorm. It is capable of performing the following actions:
+
 1. List all the open incidents on PD for a subdomain
-2. Send acknowledgment of any incident or you can even send ACK for multiple incidents using comma seprated values
-3. Close and open incident or you can even close multiple open incidents by providing comma sperated values
+2. Send acknowledgment of any incident(s)
+3. Close and open incident(s)
 4. Launch an incident by giving its details and description
 
-# Confgiuration
+# Configuration
 
-You need the following parameters to be in the config.yaml file
 `subdomain: `'name of subdomain '
```

Comment:
It would be nice if this used bullet points so it's consistent with other readmes.